src/card.tsx

```diff
@@ -211,15 +211,17 @@ const transitions: Record<VideoState, Partial<Record<VideoAction, VideoState>>>
 const videoReducer = (state: VideoState, action: VideoAction): VideoState =>
     transitions[state][action] || state;
 
+const VideoSpinner = ({size, color}: IconProps) => <Spinner size={size} color={color} delay="0" />;
+
 const getVideoActionIcon = (state: VideoState) => {
     switch (state) {
         case 'playing':
+        case 'loading':
```

Comment:
Unrelated change
Videos in cards have autoplay, so I think it makes sense that the initial icon is IconPausedFilled, otherwise we'd have a flash of a few ms showing IconPlayFilled

Reply:
If the video doesn't load fast enough, we would have a pause icon, followed by a spinner, followed by a pause icon again.

In my opinion, that is a bit weird

Reply:
I think it's weirder now in the most common case